day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,76 @@
+i = 10
+
+until i == 0
```

Comment:
Each loops are definitely a bit more canonical in ruby. How could you utilize one here?